guides/doc-Deploying_Project_on_AWS/topics/AWS_Prerequisites.adoc

```diff
@@ -59,8 +59,8 @@ In your AWS environment, complete the following steps:
 +
 [options="nowrap" subs="+quotes,attributes"]
 -----
-# {package-remove} -y rh-amazon-rhui-client*
-# {package-clean} all
+# {client-package-remove-el8} -y rh-amazon-rhui-client*
+# {project-package-clean} all
```

Comment:
Why are you mixing `project-*` and `client-*`?

Reply:
there is not attribute for "client-package-clean". I've changed project to "dnf clean" now to not confuse others.

this procedures runs on EL8+ hosts.